app/jobs/destroy_job.rb

```diff
@@ -0,0 +1,24 @@
+class DestroyJob < ApplicationJob
+  include ActivityTracking
+
+  queue_as :dradis_project
+
+  def perform(items:, author_email:, uid:)
+    # FIXME: migrate logs#uid to uuid ?
+    logger = Log.new(uid: uid)
+    logger.write do
+      "Deleting #{items.count} #{items.first.class.to_s.pluralize}"
+    end
+
+    Note.transaction do
```

Comment:
this is triggering 1 DESTROY per object, I thought we wanted to do 1 DESTROY in total?

Reply:
I wanted to call `destroy` and not `delete` so possible callbacks are called. 
If I do: Note.destroy(array_of_ids)`, looks like it also loops through the list destroying each one one by one: http://api.rubyonrails.org/classes/ActiveRecord/Relation.html#method-i-delete
Then we are good as we are? (also we are already on a background job here)